presentation/handler/invitation_handler.go

```diff
@@ -0,0 +1,134 @@
+package handler
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+	"split-the-bill-server/core"
+	. "split-the-bill-server/domain/service/service_inf"
+	"split-the-bill-server/presentation/dto"
+)
+
+// TODO: maybe delete invitationService if only one service is used.
```

Comment:
I think it's fine to keep it as it is. Maybe down the road, the InvitationHandler might need to grab some extra services, and we'd have to tack them on anyway. Doing it this way lets you easily see what stuff the handler needs just by checking out the constructor. And if there are any helper functions in there, you won't mix them up with service functions.